tests/config/test_data.config

```diff
@@ -138,17 +138,6 @@ params {
                 rnaseq_matrix                                  = "${params.test_data_base}/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.gene_counts.top1000cov.tsv"
                 deseq_results                                  = "${params.test_data_base}/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.deseq2.results.tsv"
             }
-            'illumina' {
```

Comment:
This may be because your branch is out of date- but make sure you're not messing with lines you don't intend (see also below).